### ISSUE TITLE
collision energy

### ISSUE THREAD

**Issue by HR-RH** [vdemichev/DiaNN#882]
Hello，
        Thank you for your great contribution. Will diann's output include the collision energy of precursor ions?

**Comment by vdemichev** [comment]
No, extracting this information from the raw data is at the moment not planned. What would be the purpose?

**Comment by HR-RH** [comment]
Thank you for your reply.
1. Because I want to explore whether collision energy will affect some information in secondary mass spectrometry.
2. Excuse me again, for the Fragment.Info column in report.csv, does the combination of fragment ion type and loss (e.g. y10-H2O^2/563.2714844) only include the listed types, because the output I observed only contains a single loss, will it output multiple losses (e.g. y6-H2O,-H3PO4)，that is the fragment ion loses both water molecules and phosphoric acid molecules?
